src/ChatHistory.test.tsx

```diff
@@ -2,19 +2,6 @@ import { describe, it, expect, vi, beforeEach } from 'vitest';
 import { fireEvent, render, screen } from '@testing-library/react';
 import { ChatHistory } from './ChatHistory';
 import { ConversationHistories } from './types';
-import { ChatHistoryItemProps } from './ChatHistoryItem';
-
-vi.mock('./ChatHistoryItem', () => ({
-  ChatHistoryItem: ({
-    conversation,
-    onHistoryItemClick,
-    isSelected,
-  }: ChatHistoryItemProps) => (
-    <div data-selected={isSelected} onClick={onHistoryItemClick}>
-      {conversation.title}
-    </div>
-  ),
-}));
```

Comment:
Testing clicking through the actual component


https://github.com/user-attachments/assets/767d59c3-7d55-4044-804c-4613f012cd08